src/components/link.js

```diff
@@ -79,64 +79,70 @@ const contextifyHref = (href, location, persistQuery) => {
   };
 };
 
-const Link = (props: Props) => {
-  const {
-    href: rawHref,
-    location,
-    children,
-    onClick,
-    target,
-    activeProps,
-    replaceState,
-    persistQuery,
-    push,
-    replace,
-    ...rest
-  } = props;
-
-  // Ensure the href has both a search and a query when needed
-  const normalizedHref = normalizeHref(rawHref);
-  const href = contextifyHref(normalizedHref, location, persistQuery);
-  const isActive = href.pathname === location.pathname;
-  const activeRest = (isActive && activeProps) || {};
-
-  const clickHandler = e =>
-    handleClick({
-      e,
-      target,
-      href,
+class LinkComponent extends Component {
```

Comment:
Any reason this turned into a class component? I don't see any lifecycle methods

Reply:
There was some issue with flow or lint.